INSTALL.md

```diff
@@ -21,18 +21,18 @@ git clone https://github.com/CGAL/cgal.git /path/to/cgal.git
 cd /path/to/cgal.git
 mkdir -p build/debug
 cd build/debug
-cmake -DBUILD_TYPE=Debug ../..
+cmake -DCMAKE_BUILD_TYPE=Debug ../..
 make
 ```
 
 Here is an example of how to build the library in Release:
 ``` {.bash}
-  > git clone https://github.com/CGAL/cgal.git /path/to/cgal.git
-  > cd /path/to/cgal.git
-  > mkdir -p build/debug
-  > cd build/debug
-  > cmake -DBUILD_TYPE=Release ../..
-  > make
+git clone https://github.com/CGAL/cgal.git /path/to/cgal.git
+cd /path/to/cgal.git
```

Comment:
I agree the `.git` suffix for the name of local repo could be removed. But it does not harm either.